ctapipe/containers.py

```diff
@@ -843,6 +849,19 @@ class ParticleClassificationContainer(Container):
     telescopes = Field(None, "Telescopes used if stereo, or None if Mono")
 
 
+class DispContainer(Container):
+    """
+    Standard output of disp reconstruction algorithms for origin reconstruction
+    """
+
+    default_prefix = "disp"
+
+    norm = Field(nan * u.deg, "reconstructed absolute value for disp", unit=u.deg)
```

Comment:
Why don't we just store `norm` with the sign in one float? That would use less space and still have all the information.

Reply:
What should this be called? Since `disp_parameters_disp` doesn't make much sense imo

Reply:
It's still the `norm`, it just includes the sign (as opposed to predicting dx / dy as lstchain used to do).

Reply:
Done